api/src/main/java/org/openmrs/Concept.java

```diff
@@ -1124,11 +1124,10 @@ public ConceptName getShortestName(Locale locale, Boolean exact) {
 		
 		if (locale != null) {
 			for (ConceptName possibleName : getNames()) {
-				if (possibleName.getLocale().equals(locale)) {
-					if ((shortestNameForLocale == null)
-					        || (possibleName.getName().length() < shortestNameForLocale.getName().length())) {
-						shortestNameForLocale = possibleName;
-					}
+				if (possibleName.getLocale().equals(locale)
+				        && ((shortestNameForLocale == null) || (possibleName.getName().length() < shortestNameForLocale
+				                .getName().length()))) {
+					shortestNameForLocale = possibleName;
 				}

```

Comment:
This looks to be just the same thing and my above response is ok, remembering that i created this Pull Request after checking out a new branch from master. Then pulled from upstream/gci-2014 and resolved conflicts which was the best way to confirm that every thing is updated and eliminate more conflicts